### PR TITLE
feature/issue claim from unsigned claim data

### PR DESCRIPTION
**Pull request by JGiter**
### Summary
|             |   |
|-------------|---|
| Description | As there are no more requirements on claim requester, claim can be issued from row claim data without signing it. 
| Jira issue  |  |

### CODE REVIEW

**Review by jrhender — COMMENTED** [review]
I'm not understanding the PR based on the file changes. It looks like the change is to only accept a jwt `token` instead of possibly `IPublicClaim`? But I'm not sure what that has to do with
>claim can be issued from row claim data without signing it.

**Comment by JGiter** [comment]
Purpose of this change is to free requester from signing claim before submitting to issuer. I've updated doc comment

File: packages/claims/src/claimsIssuer/claimsIssuer.ts [hunk 1]
```diff
@@ -22,11 +22,14 @@ export class ClaimsIssuer extends Claims implements IClaimsIssuer {
    * claims = new ClaimsIssuer(issuer);
    * const issuedToken = await claims.issuePublicClaim(token);
    * ```
-   * @params { string } token to verify
+   * @param { string | IPublicClaim } claim - claim to issue. Can be
+   * specified as signed or unsinged claim
```
Comment (jrhender) [inline packages/claims/src/claimsIssuer/claimsIssuer.ts:26]: Minor typo
```suggestion
   * specified as signed or unsigned claim
```

File: packages/claims/src/claimsIssuer/claimsIssuer.ts [hunk 2]
```diff
@@ -22,11 +22,14 @@ export class ClaimsIssuer extends Claims implements IClaimsIssuer {
...
    * @returns { Promise<string> } issued token
    */
-  async issuePublicClaim(token: string): Promise<string> {
-    const claim: IPublicClaim = this.jwt.decode(token) as IPublicClaim;
+  async issuePublicClaim(claim: string | IPublicClaim): Promise<string> {
```
Comment (jrhender) [inline packages/claims/src/claimsIssuer/claimsIssuer.ts:29]: Could you please add a test which issues a claim using `IPublicClaim`?

**Review by jrhender — CHANGES_REQUESTED** [review]
Makes sense now, thanks. Could you please add a test which covers this scenario?

**Review by jrhender — APPROVED** [review]
Looks good to me

**Pull request merged**